features/post/post-comment/post-comment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react';
+
+import useRelativeTime from '@/shared/hooks/useRelativeTime';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, Typography } from '@/shared/ui';
+import clsx from 'clsx';
+
+import s from './post-comment.module.scss';
+
+export type PostCommentProps = {
+  avatar?: string;
+  children?: ReactNode;
+  childrenInfoSection?: ReactNode;
+  createdAt: string;
+  iconElement?: ReactNode;
+  likesCount: number;
+  text: string;
+  textWidth?: string;
+  userName: string;
+};
+
+export const PostComment = (props: PostCommentProps) => {
+  const {
+    avatar,
+    children,
+    childrenInfoSection,
+    createdAt,
+    iconElement,
+    likesCount,
+    text,
+    textWidth,
+    userName,
+  } = props;
+  const relativeTime = useRelativeTime(createdAt);
+  const { t } = useTranslation();
+
+  const getInitials = (userName: string): string => {
+    const words = userName.split(' ');
+    const initials = words.map(word => word.charAt(0).toUpperCase());
+
+    return initials.join('');
+  };
+
+  return (
+    <div className={s.root}>
+      <div className={s.comment}>
+        <Avatar fallback={getInitials(userName)} size={'small'} src={avatar} />
+        <div className={s.body}>
+          <Typography.Regular14 className={clsx(s.body__text, textWidth)}>
+            <Typography.Bold14>{userName}</Typography.Bold14>
```

Comment:
There is a special API - `Intl.PluralRules` for automatic formation of correct multiplication. It is already used in the application as part of the `createPluralize` helper. It should be used here and below. There was a tutorial video on this topic in my personal cabinet.